docs/tools/terraform.rst

```diff
@@ -1,44 +1,39 @@
 Aiven Provider for Terraform
 =============================
 
-With Aiven's `Terraform <https://www.terraform.io>`_ provider, you can use an open-source infrastructure as code software tool to declare and manage your cloud services.
+With Aiven's `Terraform <https://www.terraform.io>`_ provider, you can use an open source infrastructure as code tool to declare and manage your cloud services.
 
-See the `Aiven Terraform provider documentation <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ to learn about the services and resources,and visit the `GitHub repository <https://github.com/aiven/terraform-provider-aiven>`_ to report any issues or contribute to the project.
+Start using Aiven Provider
```

Comment:
btw, referencing to the naming change, there will be Aiven Crossplane Provider, which is also _a provider_.

Reply:
Hashicorp suggests to stop using the word `Provider` and use `Plugin` instead, in every case, for the context of Terraform providers

cc @staceysalamon-aiven

Reply:
As this is used throughout Aiven's channels (website, etc.), I think any change like that would have to be part of a bigger discussion/project. For now, I've been trying to standardize with the Aiven Provider for Terraform name.